deploy/marketplace.csv.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   name: marketplace-operator.v0.0.1
-  namespace: marketplace
+  namespace: openshift-marketplace
```

Comment:
Should this still be in the same PR?

Reply:
Good question, the role_binding is created in `openshift-marketplace` whereas the csv was created in `marketplace`. If this isn't updated, the marketplace operator won't be deployed because of unmet dependencies.

Reply:
What are your thoughts @tkashem

Reply:
Your commit header and message should then reflect this.